lib/OpenQA/WebAPI/Plugin/Gru.pm

```diff
@@ -137,6 +139,18 @@ sub enqueue_limit_assets {
     return $self->enqueue(limit_assets => [] => {priority => 10, ttl => 172800, limit => 1});
 }
 
+sub enqueue_download_jobs {
+    my ($self, $downloads, $job_ids) = @_;
+    return unless (%$downloads and @$job_ids);
+    # array of hashrefs job_id => id; this is what create needs
+    # to create entries in a related table (gru_dependencies)
+    my @jobsarray = map +{job_id => $_}, @$job_ids;
+    for my $url (keys %$downloads) {
+        my ($path, $do_extract) = @{$downloads->{$url}};
+        $self->enqueue(download_asset => [$url, $path, $do_extract] => {priority => 20} => \@jobsarray);
```

Comment:
Where does `priority => 20` come from? Maybe a constant for this would be nice to have.

Reply:
It was there before. I just moved the code.

Reply:
I don't change the moved code here. The PR is growing too large anyways.